benchmarks/templates/benchmarks/upload.html

```diff
@@ -36,50 +36,47 @@
 <div class="container login" style="background-image: url({% static '/benchmarks/img/login_brain.png' %});">
     <div class="column is-half has-text-centered-mobile">
         <div class="container login-left">
-            {% if domain == "language"%}
-             <h1 style="font-size: 2em;">Submission System, Language</h1>
-              <br>
-              <h1 style="font-size: 1.25em;">Below you can upload a plugin. Plugins can be:</h1>
+            <h1 style="font-size: 2em;">Submission System, {{ formatted }} </h1>
+            <br>
+            <h1 style="font-size: 1.25em;">Below you can upload a plugin. Plugins can be:</h1>
             <br>
 
-              <ol style="margin-left: 0; padding-left: 1.25em;">
-                <li>A language model or revision</li>
-                <li>A benchmark or revision</li>
+            <ol style="margin-left: 0; padding-left: 1.25em;">
+                <li>A {{ domain }} model or revision</li>
+                <li>A {{ domain }} benchmark or revision</li>
                 <li>A dataset or revision</li>
                 <li>A metric or revision</li>
```

Comment:
be consistent, either list `{{ domain }}` for all plugin types or none. (don't think you have to)